### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by im-n1** [BasioMeusPuga/Lector#106]
![image](https://user-images.githubusercontent.com/198654/60769928-e3182780-a0d5-11e9-8b5d-d22256a19ac6.png)

**Comment by BasioMeusPuga** [comment]
Yeah, `pip` and Virualenv installs cause errors because of that line. You're okay to remove `single-version-externally-managed=1` from `setup.cfg`, but afaik, I have to keep the line in master to not have everything spread out across the python `site-packages` directory.

**Comment by im-n1** [comment]
Is there a way how to incorporate both ways withourlt dirty line removal?
